src/cli/library_install.js

```diff
@@ -46,15 +46,24 @@ class CLILibraryInstallCommandSite extends LibraryInstallCommandSite {
 	}
 
 	async notifyCheckingLibrary(libName){
+		if (this.argv.json){
+			return;
+		}
```

Comment:
Oof the need for a central logging/reporting setup is so strong.

Reply:
yes indeedy!